hive/src/test/java/org/apache/iceberg/hive/HiveCatalogBaseTest.java

```diff
@@ -156,7 +165,7 @@ private static TServer thriftServer() throws Exception {
             .transportFactory(new TTransportFactory())
             .protocolFactory(new TBinaryProtocol.Factory())
             .minWorkerThreads(3)
-            .maxWorkerThreads(5);
+            .maxWorkerThreads(32);
```

Comment:
What was the reason for changing this? Keeping it low helps us catch client leaks, like the one introduced by the `newClient` call in the Hive catalog.